lib/db.js

```diff
@@ -3,6 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 var P = require('bluebird')
+var extend = require('util')._extend;
```

Comment:
In many ways I'd prefer the `.account()` and `.emailRecord()` to just return certain fields explicitly rather than taking the entire account and filtering the `verifyHash` out of it.

This also means we don't need these extra two functions which live somewhat on the outside of the `Memory` class (even though our data stores also live outside of them). I guess that makes this point defunct - so I'll just leave it to the point above for discussion.